### ISSUE TITLE
[BUG] Uneven spacing in title bar for tabs

### ISSUE THREAD

**Issue by maxwolv** [AdamXweb/WhiteSurFirefoxThemeMacOS#103]
### Describe the problem

The title bar for my tabs are uneven. I have deleted Firefox and reinstalled, then did a clean install of WhiteSurFirefox. I believe it is something related to a custom folder that Firefox is pulling from? Unsure of how to resolve.

### Version

1.5.3

### OS

macOS 13.5.2

### Firefox Version

117.01

### Install Options

bash ./install.sh -c -n -s -e

### Colour Theme

no

### Additional info

![Screenshot 2023-09-26 at 13 20 28](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/assets/94148437/eb2ea47c-e9d2-43d8-a2d2-4c09a4971626)

here is a screenshot to show what I mean.

### Please tick the boxes

- [X] You are using a [current](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/) version of the theme. (check the date associated with the version)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/adamxweb/WhiteSurFirefoxThemeMacOS/issues?q=is%3Aissue)

**Comment by AdamXweb** [comment]
Looks like this is related to the `-c` install flag to move the tab close button to the left.
Will need to investigate further when I have time

**Comment by dcd-arnold** [comment]
1. Is this a duplicate of #87?
2. Maybe [this commit](https://github.com/rafaelmardojai/firefox-gnome-theme/commit/8c9db7a9cdd2012868e0fc97b329d1f2a7af9823) may help?

**Comment by dcd-arnold** [comment]
@AdamXweb this should be resolved now after #124

**Comment by AdamXweb** [comment]
@dcd-arnold Thanks for the pickup - closing issue
